### PR TITLE
PROTO-1782: plugin setup

### PR DESCRIPTION
**Pull request by alecsavvy**
the filepath / kv map was a bit overly complicated especially if we're just going to use that file path anyway so went with this quick method

### CODE REVIEW

File: pkg/conf/model.go [hunk 1]
```diff
@@ -128,3 +130,8 @@ type Infra struct {
 	AWSSecretAccessKey string `yaml:"awsSecretAccessKey,omitempty"`
 	AWSRegion          string `yaml:"awsRegion,omitempty"`
 }
+
+
+// name of the plugin that should match with it's docker profile
+// this will be used with the REGISTER_PLUGINS env var
+type PluginName = string
```
Comment (phelpsdb) [inline pkg/conf/model.go:137]: What's the difference between `type PluginName = string` and `type PluginName string`?

File: pkg/conf/override.go [hunk 2]
```diff
@@ -54,5 +56,16 @@ func (config *NodeConfig) ToOverrideEnv(host string, nc NetworkConfig) map[strin
 		overrideEnv[k] = v
 	}
 
+	// register plugins and pass their configs into
+	// override .env
+	plugins := []string{}
```
Comment (phelpsdb) [inline pkg/conf/override.go:61]: What's the difference between `[]string{}` and `[]string`?

**Review by phelpsdb — APPROVED** [review]
Pretty straightforward, just some qs about unfamiliar syntax

File: pkg/conf/model.go (hunk 1 above)
Comment (phelpsdb) [reply pkg/conf/model.go:137]: I asked chatgpt and got an answer, I guess the real question is whether it matters if we're using one form or the other given that other type declarations in the file omit the `=`

File: pkg/conf/override.go (hunk 2 above)
Comment (phelpsdb) [reply pkg/conf/override.go:61]: Resolved, for some reason I thought this was a declaration and not an assignment making it an empty slice literal.

File: pkg/conf/model.go (hunk 1 above)
Comment (alecsavvy) [reply pkg/conf/model.go:137]: Ah so i made a type alias so PluginName is just a string but when reading the code it's a bit more explicit. The latter is a new type so if we wanted to add methods to PluginName alongside methods that string already has we could do it that way

Comment (alecsavvy) [reply pkg/conf/model.go:137]: in this case there's not a huge diff

File: pkg/conf/override.go (hunk 2 above)
Comment (alecsavvy) [reply pkg/conf/override.go:61]: Yep, i could've done `var plugins []string` and i think got the same result. Sometimes default go types I forget how they work so I do the assignment as it's a bit more predictable.

Comment (alecsavvy) [reply pkg/conf/override.go:61]: I'm not sure which one is exactly more idiomatic but both work

**Comment by alecsavvy** [comment]
tested with `./audius-ctl` and alec sandbox, ran both trending rewards and notifications (without full config) but they start up with env vars correctly

**Pull request merged**